docs/index.trst

```diff
@@ -19,3 +19,11 @@ Revision: |release|
     articles/templates/index.rst
 
     articles/workflow/git-workflow.rst
+
+.. toctree::
+	:caption: Categories:
+	:maxdepth: 1
+
+	categories/howto.rst
+
+    categories/processes.rst
```

Comment:
Is this supposed to be indented like this? Shouldn't it be part of the toctree?

Reply:
No, you are correct. I also just noticed this and fixed it.
Looks like my editor used tabs instead of spaces.